src/cli/config/bundler.ts

```diff
@@ -83,7 +84,20 @@ export const bundlerArgsSchema = z.object({
     "max-gas-per-bundle": z
         .string()
         .transform((val) => BigInt(val))
-        .default("5000000")
+        .default("5000000"),
+    "supported-rpc-methods": z
```

Comment:
Can we say enabled-rpc-methods instead of supported? As we do support all? Also default can be nullish that means all are enabled?

Reply:
or actually just `rpc-methods` and `refilling-wallets` @kristofgazso what do you think? 

rest lgtm